packages/teams-js/test/public/media.spec.ts

```diff
@@ -815,9 +688,11 @@ describe('media', () => {
         expect(mediaAttachment.size).not.toBeNull();
         expect(typeof mediaAttachment.size === 'number').toBeTruthy();
         expect(mediaAttachment.getMedia).toBeDefined();
+        await expect(promise).resolves;
```

Comment:
I think .resolves takes a matcher like .toBe or something the promise that is expected to resolve with some result.
So line 691 is redundant and doesn't test anything.